lib/src/model/explorer/tablebase.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:deep_pick/deep_pick.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'tablebase.freezed.dart';
+part 'tablebase.g.dart';
+
+enum TablebaseCategory {
+  win,
+  unknown,
+  syzygyWin,
+  maybeWin,
+  cursedWin,
+  draw,
+  blessedLoss,
+  maybeLoss,
+  syzygyLoss,
+  loss,
+}
+
+@Freezed(fromJson: true)
```

Comment:
The `fromJson` is not necessary here, since you're implementing it yourself using `fromPick`.